uploader.json

```diff
@@ -1005,6 +1005,26 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_d"
         },
+        {
+            "file": "/opt/rh/httpd24/root/etc/httpd/conf/()*httpd\\.conf",
+            "pattern": [],
+            "symbolic_name": "httpd_conf_scl_httpd24"
+        },
+        {
+            "file": "/opt/rh/httpd24/root/etc/httpd/conf.d/()*.+\\.conf",
+            "pattern": [],
+            "symbolic_name": "httpd_conf_d_scl_httpd24"
```

Comment:
@matysek please use the same `symbolic_name` name as the spec for both of these since they both derive from one spec in insights-core.  Please do the same for `httpd_conf_scl_jbcs_httpd24`.